high_card.rb

```diff
@@ -1,5 +1,56 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K","A"]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+deck = []
+
+#make deck
+ranks.each do |rank|
+  suits.each do |suit|
+    deck.push([rank,suit])
+  end
+end
+
+#shuffle deck
+deck = deck.shuffle
+
+# prompt players
+players = []
+userInput = nil
+
+puts players.size.to_s + " players so far. Enter a player name (no limit on amount of players), or type 'play'"
+while !(userInput.to_s == "play")
```

Comment:
I think this is minutia, but another way to write this is `userInput.to_s != "play"`